tests/test_subscription.py

```diff
@@ -30,6 +30,16 @@ def sub_published_time_type(data, **kwargs):
     logger.info(f'{type(kwargs["published_at"])}')
 
 
+def landscape_filter(**kwargs):
+    return kwargs.get('type') == 'landscape'
+
+
+@sub(topic='photo-updated', filter_by=landscape_filter)
+def sub_process_landscape_photos(data, **kwargs):
+    photo_type = kwargs.get('type')
+    logger.info(f'Received a photo of type {photo_type}')
```

Comment:
I'd like to suggest to return a value to avoid inspecting log messages

Reply:
Much better, thanks!